cmd/cnab-to-oci/pull.go

```diff
@@ -40,6 +42,20 @@ func runPull(opts pullOptions) error {
 	if err != nil {
 		return err
 	}
+
+	if opts.raw {
+		index, err := remotes.GetIndex(context.Background(), ref, createResolver(opts.insecureRegistries).Resolver)
+		if err != nil {
+			return err
+		}
+
+		bytes, err := json.MarshalIndent(index, "", "\t")
+		if err != nil {
+			return err
+		}
+		return ioutil.WriteFile(opts.output, bytes, 0644)
```

Comment:
I think it should handle the stdout output as well. We can factorize with something like this:
```go
func writeFile(output string, data []byte) error{
    if output == "-" {
        fmt.Fprintln(os.Stdout, string(data))
        return nil
    }
    return ioutil.WriteFile(output,data, 0644)
}
```